lib/cdo/script_constants.rb

```diff
@@ -51,14 +56,43 @@ def ScriptConstants.hoc?(name)
         MINECRAFT_NAME,
         INFINITY_NAME,
         ARTIST_NAME
```

Comment:
Please confirm these changes don't impact the gsheet. @tanyaparker might be the person to ask.